components/Slider/tests/unit/SliderTests.m

```diff
@@ -1225,26 +1225,83 @@ - (void)testEnabledFullHapticFeedback {
   self.slider.minimumValue = 0;
   self.slider.maximumValue = 5;
   self.slider.hapticsEnabled = YES;
-  self.slider.numberOfDiscreteValues = 2;
+  self.slider.numberOfDiscreteValues = 6;
   self.slider.shouldEnableHapticsForAllDiscreteValues = YES;
 
   if (@available(iOS 10.0, *)) {
     _mockFeedbackGenerator = [[MockUIImpactFeedbackGenerator alloc] init];
     self.slider.feedbackGenerator = _mockFeedbackGenerator;
     for (NSUInteger i = 0; i < 6; ++i) {
-      self.slider.value = i;
+      //self.slider.value = i;
+      [self.slider setValue:i];
```

Comment:
Is the above line supposed to be commented out or deleted?  I'd prefer we set the property instead of passing the message.